<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ml_remapper_selector</class>
 <widget class="QWidget" name="ml_remapper_selector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>52</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Remappers</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cmb_remapper_file">
       <property name="toolTip">
        <string>Transform</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wdg_buttons" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="btn_transform_only">
          <property name="toolTip">
           <string>Transform Only</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="slim5.qrc">
            <normaloff>:/images/paint-splash.svg</normaloff>:/images/paint-splash.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_phase_only">
          <property name="toolTip">
           <string>Phase Only</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="slim5.qrc">
            <normaloff>:/images/paint-splash_outline.svg</normaloff>:/images/paint-splash_outline.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_overlay">
          <property name="toolTip">
           <string>Overlay</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="slim5.qrc">
            <normaloff>:/images/paint-splash_plus.svg</normaloff>:/images/paint-splash_plus.svg</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="ml_display_settings" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="cmb_luts">
        <property name="toolTip">
         <string>Transform Lut</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="min_range">
        <property name="toolTip">
         <string>Transform Min</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-131072.000000000000000</double>
        </property>
        <property name="maximum">
         <double>131072.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="max_range">
        <property name="toolTip">
         <string>Transform Max</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="minimum">
         <double>-131072.000000000000000</double>
        </property>
        <property name="maximum">
         <double>131072.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="slim5.qrc"/>
 </resources>
 <connections/>
</ui>
